### PR TITLE
WIP: Challenge storing groceries

### PR DESCRIPTION
**Pull request by jlunenburg**

### CODE REVIEW

File: challenge_manipulation/src/manipulation.py [hunk 1]
```diff
@@ -665,7 +665,7 @@ def export_to_pdf(userdata):
             entities = [ e[0] for e in DETECTED_OBJECTS_WITH_PROBS ]
 
             # Export images (Only best MAX_NUM_ENTITIES_IN_PDF)
-            pdf.entities_to_pdf(robot.ed, entities[:MAX_NUM_ENTITIES_IN_PDF], "tech_united_manipulation_challenge")
+            # pdf.entities_to_pdf(robot.ed, entities[:MAX_NUM_ENTITIES_IN_PDF], "tech_united_manipulation_challenge")
```
Comment (LoyVanBeek) [inline challenge_manipulation/src/manipulation.py]: Remember to uncomment this later

File: challenge_storing_groceries/scripts/add_entities.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+import roslib; roslib.load_manifest('fast_simulator')
```
Comment (LoyVanBeek) [inline challenge_storing_groceries/scripts/add_entities.py]: Huh?

File: challenge_storing_groceries/src/challenge_storing_groceries/test_empty_shelf_designator.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+USE_SLAM = True  # Indicates whether or not to use SLAM for localization
+if USE_SLAM:
+    CABINET = challenge_knowledge.cabinet_slam
+else:
+    CABINET = challenge_knowledge.cabinet_amcl
+
+PLACE_SHELF = challenge_knowledge.place_shelf
+
+'''
```
Comment (LoyVanBeek) [inline challenge_storing_groceries/src/challenge_storing_groceries/test_empty_shelf_designator.py]: This sort of comments are hard to see in a diff. Just use a block of #'s plz

File: challenge_storing_groceries/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(challenge_storing_groceries)
+
+find_package(catkin REQUIRED COMPONENTS
+    rospy
+    robot_smach_states
+    robot_skills)
```
Comment (Rayman) [inline challenge_storing_groceries/CMakeLists.txt]: Its not a BUILD dep, only run.

**Pull request merged**